sonar-project.properties

```diff
@@ -0,0 +1,20 @@
+
+
```

Comment:
Not overly bothered but the whitespace could be removed.

Reply:
done